archive-product.php

```diff
@@ -11,33 +11,31 @@
     <div class="row margin-bottom-small">
 <?php
 if( have_posts() ) {
-  $i = 0;
   while( have_posts() ) {
     the_post();
 
     $product = new WC_Product($post->ID);
     $in_stock = $product->is_in_stock();
     $availability = $product->get_availability();
 
-    if ($i % 4 === 0 && $i !== 0) {
-      echo '</div>\n<div class="row margin-bottom-small">';
```

Comment:
ok sweet flex deals with the flow thing nicely:

![screen shot 2016-08-12 at 16 35 54](https://cloud.githubusercontent.com/assets/845186/17637687/e1e29c56-60aa-11e6-915a-d6af405af11a.png)